exporters/otlp/trace/build.gradle

```diff
@@ -10,7 +10,7 @@ description = 'OpenTelemetry Protocol Trace Exporter'
 ext.moduleName = "io.opentelemetry.exporter.otlp.trace"
 
 dependencies {
-    api project(':sdk:all')
+    api project(':sdk:trace')
```

Comment:
This needs only implementation dependency not api correct?

Reply:
Actually it could be runtime dependency in this case.

Reply:
since the exporter implements an interface in the sdk:trace module, it effectively exposes that dependency as a part of it's API surface, so this has to be an `api` dependency, I believe.

Reply:
Sorry I read this metrics :))